packages/browser/src/domain/configuration.ts

```diff
@@ -46,6 +47,16 @@ export interface FlaggingInitConfiguration extends InitConfiguration {
    */
   enableExposureLogging?: boolean
 
+  /**
+   * Whether to enable flag evaluation tracking via the flag evaluation intake
+   */
+  enableFlagEvaluationTracking?: boolean
+
+  /**
+   * Flag evaluation tracking interval in milliseconds (default: 10000ms)
+   */
+  flagEvaluationTrackingInterval?: number
```

Comment:
Where is the default for this set?

edit: i think it is from `DEFAULT_FLAG_EVALUATION_TRACKING_INTERVAL`. IMO it would be cleaner to have the default set here in this object rather than the downstream. having the default only set in a comment can lead to it getting out of sync with the actual default, and it makes changing the default more difficult as one needs to trace through all of the code.